aws/cloudformation/iam.yml.erb

```diff
@@ -665,6 +673,27 @@ roles.each do |name, config| %>
       Value: initial,value
       Description: List of Google IDs of people being granted billing access.
 
+  AthenaCloudwatchLogsQueryPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      ManagedPolicyName: AthenaCloudwatchLogsQueryPolicy
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - athena:StartQueryExecution
+              - athena:GetQueryExecution
+            Resource: !Sub 'arn:aws:athena:${AWS::Region}:${AWS::AccountId}:workgroup/primary'
+          - Effect: Allow
+            Action:
+              - s3:GetObject
+              - s3:PutObject
+              - s3:ListBucket
+            Resource: 
+              - !Sub “arn:aws:s3:::aws-athena-query-results-${AWS::AccountId}-${AWS::Region}/*”
+              - !Sub “arn:aws:s3:::aws-athena-query-results-${AWS::AccountId}-${AWS::Region}”
```

Comment:
I believe this is a default naming scheme, but it is in fact the naming scheme of the current production s3 bucket used for this purpose.